bin/check-rds.rb

```diff
@@ -161,7 +161,7 @@ def find_db_instance(id)
 
   def find_db_cluster_writer(id)
     wr = rds.describe_db_clusters(db_cluster_identifier: id).db_clusters[0].db_cluster_members.detect(&:is_cluster_writer).db_instance_identifier
-    unknown 'DB cluster not found.' if cl.nil?
+    unknown 'DB cluster not found.' if wr.nil?
```

Comment:
I really dislike short undescriptive variable names like this, I know its outside of the scope of change...I will probably go back in afterwards and fix this up.